pkgs/development/python-modules/busylight-for-humans/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pytestCheckHook,
+  pytest-mock,
+  bitvector-for-humans,
+  hidapi,
+  loguru,
+  pyserial,
+  typer,
+  webcolors,
+}:
+buildPythonPackage rec {
+  pname = "busylight-for-humans";
+  version = "0.28.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "JnyJny";
+    repo = "busylight";
+    rev = version;
+    hash = "sha256-OGcq+mrjaKUIAyLR8tknX+I5EDmVCBVZE406K4+PaWc=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail 'poetry>=0.12' 'poetry-core' \
+      --replace-fail 'poetry.masonry.api' 'poetry.core.masonry.api'
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+    pytestCheckHook
+    pytest-mock
+  ];
+  propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```

Newer lingo for propagatedBuildInputs in python packages.